guilded/embed.py

```diff
@@ -18,7 +19,7 @@ def __len__(self):
         return len(self.__dict__)
 
     def __repr__(self):
-        return 'EmbedProxy(%s)' % ', '.join(('%s=%r' % (k, v) for k, v in self.__dict__.items() if not k.startswith('_')))
+        return 'EmbedProxy(%s)' % ', '.join(['%s=%r' % (k, v) for k, v in self.__dict__.items() if not k.startswith('_')])
```

Comment:
Why?

Reply:
This is faster, str.join needs to know length of the iterable. Generator results in wasting cpu